hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -148,33 +168,65 @@ class RobotHW : public InterfaceManager
     ERROR
   };
 
-  // Return (in realtime) the state of the last doSwitch()
+  /** \brief Return (in realtime) the state of the last doSwitch() */
   virtual SwitchState switchResult() const
   {
     return DONE;
   }
 
-  // Return (in realtime) the state of the last doSwitch() for a given controller
+  /** \brief Return (in realtime) the state of the last doSwitch() for a given controller */
   virtual SwitchState switchResult(const ControllerInfo& /*controller*/) const
   {
     return DONE;
   }
+  /**\}*/
 
-  /**
-   * Reads data from the robot HW
+  
+  /** \name Control Loop
+   * 
+   */
+   /**\{*/
```

Comment:
Should this be the same as the other groups? I'm not a Doxygen expert, just looking at consistency. Maybe this has something to do with the different grouping you see when you generate the doc? No idea :)

```suggestion
  /** \name Control Loop
   *\{*/
```

Reply:
Thanks for the hint! Unfortunately it still looks different, not sure how to fix this.

Reply:
Oh, I think I figured it out. The `Hardware Interface Switching` group contains the `SwitchState` enum, which isn't a member func, so the group gets displayed differently. Take a look at [this page of the Doxygen manual](http://www.doxygen.nl/manual/grouping.html#memgroup). near the bottom.

Not sure whether it's better to leave the enum as is, or move it out of the group so that it's more logical in the generated HTML... Let's get other opinions.

In either case, let's update the group definitions anyways for consistency. From the manual, it looks like the name is supposed to be in a different comment than the `\{`, but we've been getting on fine with them in the same comment, so let's stick to that.